sdk/storage/Azure.Storage.Common/src/Shared/StorageResponseClassifier.cs

```diff
@@ -43,5 +43,30 @@ public override bool IsRetriableResponse(HttpMessage message)
             }
             return base.IsRetriableResponse(message);
         }
+
+        /// <inheritdoc />
+        public override bool IsErrorResponse(HttpMessage message)
+        {
+            switch (message.Response.Status)
+            {
+                case 409:
+                    var header = message.Request.Headers;
```

Comment:
Nit - rename as `headers` and isn't this a violation of our `var` rules?

Reply:
VAR ALL THE THINGS!!!!